src/Confluent.Kafka.Avro/AvroDeserializer.cs

```diff
@@ -14,14 +14,15 @@
 //
 // Refer to LICENSE for more information.
 
+using System.Threading.Tasks;
 using System.Collections.Generic;
 using Avro.Generic;
 using Confluent.SchemaRegistry;
 using System;
 using System.Linq;
 
 
-namespace Confluent.Kafka.Serialization
+namespace Confluent.Kafka.AvroClients
 {
```

Comment:
Ideally, I would have liked to put the avro consumer, producer and serdes in the namespace `Confluent.Kafka.Avro`, however this makes usage inconvenient when also referencing the the Apache Avro which puts all functionality in the `Avro` namespace (the situation would be better if they had have used `Apache` at the top level instead). `Confluent.Kafka.AvroSerdes` is the best I came up with. I also considered `Confluent.Kafka.AvroClients`, and just `Confluent.Kafka`.